examples/Bremsstrahlung/include/simulation_defines/param/bremsstrahlung.param

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
@ax3l removed the doxygen brief for license header in #1841, could you please fix it. #1841 contains the sed script to remove it.